agent/mission_solver/__main__.py

```
@@ -0,0 +1,99 @@
+    print(f"\n候选副本: {len(quests)} 个")
+    if args.progress > 0:
+        print(f"  (主线进度限制: warId <= {args.progress})")
+
+    # 求解
+    print("\n求解中...")
+    result = solve(quests, missions)
+
+    if not result.plan:
+        print("\n无法找到可行方案。可能原因:")
+        print("  - 部分任务条件无法被任何副本满足")
+        print("  - 数据不完整，请更新副本数据")
```

Ruff (0.15.11)

[warning] String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)